tests/include/catch.hpp

```diff
@@ -10749,7 +10749,7 @@ namespace {
     }
 }
 
-#endif // signals/SEH handling
+#endif // signals/SHE handling
```

Comment:
I think these should still be SEH (all in this file)

Reply:
yep, I will see why it didn't get ignored - it was on my list